source/examples/verovio/MoszkowskiStudy.mei

```diff
@@ -0,0 +1,138 @@
+<?xml-model href="https://music-encoding.org/schema/dev/mei-CMN.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0" ?>
+<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.1-dev">
+  <meiHead>
+    <fileDesc>
+      <titleStmt>
+        <title>Moszkowski’s 12 Pianoforte Studies for the left hand (excerpt)</title>
+        <editor>
+          <persName>Klaus Rettinghaus</persName>
+        </editor>
+      </titleStmt>
+      <pubStmt>
+        <date>2024</date>
+        <availability>
+          <useRestrict label="licence" auth="https://creativecommons.org/licenses/by/" codedval="4.0">
+            <p>Creative Commons Attribution 4.0 International License</p>
+          </useRestrict>
+        </availability>
+      </pubStmt>
+      <seriesStmt>
+        <title>MEI Guidelines examples</title>
+      </seriesStmt>
+    </fileDesc>
+    <workList>
+      <work>
+        <title>Moszkowski Study No. 4 (excerpt)</title>
+      </work>
+    </workList>
+  </meiHead>
+  <music>
+    <body>
+      <mdiv n="4" label="Moszkowski">
+        <score>
+          <scoreDef spacing.staff="16" pedal.style="altpedstar" mnum.visible="false">
+            <staffGrp symbol="brace">
+              <staffDef n="1" clef.line="2" clef.shape="G" lines="5">
+                <keySig sig="1s" />
+              </staffDef>
+              <staffDef n="2" clef.line="4" clef.shape="F" lines="5">
+                <keySig sig="1s" />
+              </staffDef>
+            </staffGrp>
+          </scoreDef>
+          <section>
+            <measure n="35" right="single">
+              <staff n="1">
+                <layer>
+                  <mSpace />
+                </layer>
+              </staff>
+              <staff n="2">
+                <layer>
+                  <beam place="above">
+                    <note dur="16" oct="1" pname="b" xml:id="ex-35-01" />
+                    <note dur="16" oct="2" pname="f" accid.ges="s" />
+                    <note dur="16" oct="2" pname="b" />
+                    <note dur="16" oct="3" pname="d">
+                      <accid accid="s" />
+                    </note>
+                  </beam>
+                  <beam place="below">
+                    <note dur="16" oct="3" pname="f" accid.ges="s" />
+                    <note dur="16" oct="3" pname="b" />
+                    <note dur="16" oct="4" pname="d" staff="1">
+                      <accid accid="s" />
+                    </note>
+                    <note dur="16" oct="4" pname="f" accid.ges="s" staff="1" />
+                  </beam>
+                  <?edit-start?>
+                  <beam place="mixed">
+                    <note dur="16" oct="4" pname="b" staff="1" />
+                    <note dur="16" oct="4" pname="f" accid.ges="s" staff="1" />
+                    <note dur="16" oct="4" pname="d" accid.ges="s" staff="1" />
+                    <note dur="16" oct="3" pname="b" staff="2" />

```

Comment:
Not really, but it doesn't hurt, does it?

Reply:
I guess not, but it depends what we want to present as "best practice"?

Reply:
I think it would be better not to have it

Reply:
Then we'll have to present the full staff element, which is exceeds the needs here.

Reply:
If you think we need to present the full staff element and that it is too much to have it, then it would probably be preferable to make a shorter example.

Reply:
relates to #713